app/models/review.rb

```diff
@@ -1,4 +1,11 @@
 class Review < ActiveRecord::Base
+  RATING = [
```

Comment:
This is actually an interesting violation that we haven't really dealt with yet: This is suggesting that you "Freeze" the array, by calling `'.freeze` on the end of it. 

I encourage you to look into the `freeze` method, and then consider whether or not you agree with this warning. 

If you agree with it, then fix the code and resubmit. 

If you disagree with it, then remove the hound rule, and resubmit.